scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -231,8 +241,11 @@
            base-path
            google-credentials
            verifying-ssl
-           bearer-token-refresh-seconds]
-    :or {bearer-token-refresh-seconds 300}}
+           bearer-token-refresh-seconds
+           namespace]
+    :or {bearer-token-refresh-seconds 300
```

Comment:
Can we put documentation for these kubernetes compute cluster options in the config file documentation? 
(I'm not asking you to document for every option, but can you document this option and its possible values?)

Reply:
Done